jobManager.py

```diff
@@ -82,6 +82,9 @@ def __manage(self):
         elif Config.VMMS_NAME == "ec2SSH":
             from vmms.ec2SSH import Ec2SSH
             vmms = Ec2SSH()
+        elif Config.VMMS_NAME == "dockerSSH":
+            from vmms.dockerSSH import DockerSSH
+            vmms = DockerSSH()
 
```

Comment:
If system needs to exit, I think there should be one Rescue here that catches all the initialization errors, not just docker, and exit.